nntrainer/utils/node_exporter.cpp

```diff
@@ -37,11 +39,26 @@ namespace nntrainer {
 constexpr const unsigned int CONV2D_DIM = 2;
 constexpr const unsigned int POOLING2D_DIM = 2;
 
+#ifdef ENABLE_TFLITE_INTERPRETER
+
+/**
+ * @brief Construct a new Exporter object
+ *
+ */
+Exporter::Exporter() :
+  fbb(nullptr),
+  stored_result(nullptr),
+  is_exported(false) {}
```

Comment:
How about making parent class of Exporter and making tflite_node_exporter inheriting the Exporter?